Source/JavaScriptCore/API/JSValue.mm

```diff
@@ -1284,8 +1284,8 @@ - (JSValue *)initWithValue:(JSValueRef)value inContext:(JSContext *)context
         if (strcmp(idType, "@") != 0)
             return;
         {
-            auto type = adoptSystemMalloc(method_copyArgumentType(method, 2));
-            structHandlers->add(StringImpl::createFromCString(type.get()), (StructTagHandler) { selector, 0 });
+            String type = byteCast<Latin1Character>(unsafeSpan(adoptSystemMalloc(method_copyArgumentType(method, 2)).get()));
```

Comment:
I think the two line version of this is a bit more clear. From reading this, it isn't immediately clear to me why the result adoptSystemMalloc() isn't destroyed before its used.